include/net/nrf_cloud_defs.h

```diff
@@ -180,9 +184,9 @@
 #define NRF_CLOUD_JSON_KEY_APP_VER		"appVersion"
 #define NRF_CLOUD_JSON_VAL_NOT_ASSOC		"not_associated"
 #define NRF_CLOUD_JSON_VAL_PAIRED		"paired"
-/** Current FOTA version string used in device shadow */
+/* Current FOTA version string used in device shadow */
 #define NRF_CLOUD_FOTA_VER_STR			"fota_v" STRINGIFY(NRF_CLOUD_FOTA_VER)
-/** Max length of nRF Cloud's stage/environment name */
+/* Max length of nRF Cloud's stage/environment name */
```

Comment:
Are these asterisk deletions intended to be in this PR?

If so, should the two lines following them also be updated?

Reply:
intentional... i didn't feel these needed doc comments.
the later ones are more useful imo.